### PR TITLE
Fix missing unit characters on zero data widgets.

### PR DESCRIPTION
**Pull request by techanvil**
## Summary

Addresses issue #4238

## Relevant technical choices

It turns out that the only stories currently in our VRT test suite which have a relevant zero-data variant (i.e. one which shows `DataBlock` components with zero data) are for the Search Console and Analytics Overview widgets.

Other widgets/components currently in the VRT suite which use `DataBlock` show a descriptive view when there is zero data, for example the AdSense Overview widget, which displays the text "No ad impressions yet": http://localhost:9001/?path=/story/adsense-module-components-module-overview-widget--data-unavailable.

Therefore, I have only added the zero-data Search Console and Analytics Overview widgets to the VRT test suite.

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: .storybook/storybook-data.js [hunk 1]
```diff
@@ -95,6 +95,22 @@ module.exports = [
 			},
 		},
 	},
+	{
```
Comment (eugene-manuilov) [inline .storybook/storybook-data.js:98]: Thanks, @techanvil. This PR looks good to me, but could you please add a VR test for the zero-data story of the Overview widget in the AdSense module too to make sure that all overview widgets have their zero-data states covered? https://github.com/google/site-kit-wp/blob/0f7293812b3c9a1ce2f023c2cb70c31becf38fc1/.storybook/storybook-data.js#L68-L82

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]

File: .storybook/storybook-data.js (hunk 1 above)
Comment (techanvil) [reply .storybook/storybook-data.js:98]: Hi @eugene-manuilov, thanks for the review. I discovered during implementation that the AdSense Overview widget does not have a zero-data story in the same way the Search Console and Analytics Overview widgets do. I did mention this in the **Relevant technical choices** section at the top of this PR.

image

I actually tried adding a **Zero Data** story, but discovered that the AdSense Overview widget renders the view shown in the story **Data Unavailable** when the data is zeroed, due to these lines:

https://github.com/google/site-kit-wp/blob/8bb0a20a1306d8ad4f598f377b528c970bbcbd47/assets/js/modules/adsense/components/module/ModuleOverviewWidget/index.js#L143-L155

So, I figured I'd simply leave the AdSense Overview widget out of this PR. I would however be happy to revisit this choice. What do you think, given the above?

Comment (eugene-manuilov) [reply .storybook/storybook-data.js:98]: Ah... sorry, I didn't notice your comment in the relevant technical choices section... :sweat_smile: That makes sense then. Let's not add it for the AdSense module. :+1:

**Review by eugene-manuilov — APPROVED** [review]
LGTM

File: .storybook/storybook-data.js (hunk 1 above)
Comment (techanvil) [reply .storybook/storybook-data.js:98]: No problem, and thanks for the merge!
